docs/creating_your_tooltip.md

```diff
@@ -0,0 +1,77 @@
+# Creating Your Tooltip
+
+##### Step 1. Add The Library
+
+Include the `vg-tooltip` library and its dependencies (`d3`, `vega`, `vega-lite`, `vega-embed` and `datalib`) in your HTML `<head>`.
```

Comment:
Add least add a line to suggest that it's better to use npm to install dependency
just like http://vega.github.io/vega-lite/usage/embed.html